### ISSUE TITLE
Demo page is out of date

### ISSUE THREAD

**Issue by JeromeMartinez** [MediaArea/MediaConch-Website#80]
https://mediaarea.net/MediaConch/demo.html

Still XSL inside, but this is no more (well) supported and should not be advised.
And links are dead.

**Comment by dericed** [comment]
oh this page is so old. I'll check if anything here is worthwhile to save.

**Comment by ablwr** [comment]
Yeah we unlinked it from the navigation I think in hopes of it being restored

**Comment by JeromeMartinez** [comment]
> we unlinked it from the navigation

Ho, I missed that. So it is "only" in source code.
I keep the ticket open in order to decide in the future what we do (delete the page from source code or reuse/relink it after modifications)

**Comment by ablwr** [comment]
Yeah... I didn't want to delete it entirely because "Remake the demo" has been tasked to several people in the past but not to me, and I didn't know if they would need to reference this one when making a new one.
